3-UsingAPIs/Week2/homework/ex6-browserDebug/index.js

```diff
@@ -39,7 +49,7 @@ function renderLaureates(laureates) {
 
 async function fetchAndRender() {
   try {
-    const laureates = getData(
+    const { laureates } = await getData(
```

Comment:
👍  Good work. I hope you learned about the JS debugger - it is very useful tool!